packages/clarity-native-bin/src/directInstall.ts

```diff
@@ -13,7 +13,8 @@ export default (async () => {
       process.exit();
     }
   } catch (error) {
-    console.error(`Failed to install clarity-cli native binary: ${error}`);
+    console.error(`Failed to install clarity-cli native binary:`);
+    console.error(error);
```

Comment:
`yarn install` has some bug where it cuts off long error messages during the "running build scripts" stage. This helps show the full error and debug.